lib/s3_cors_rulesets.rb

```diff
@@ -26,6 +26,10 @@ class S3CorsRulesets
     max_age_seconds: 3000
   }.freeze
 
+  RULE_STATUS_SKIPPED = "rules_skipped_from_settings".freeze
+  RULE_STATUS_EXISTED = "rules_already_existed".freeze
+  RULE_STATUS_APPLIED = "rules_applied".freeze
```

Comment:
`# frozen_string_literal: true` makes these freezes redundant, doesn't it?

Reply:
:cold_face: :cold_face: :cold_face: yes it does, I will remove :) Forget about that comment, I just add it mindlessly now.